doc/classes/ProjectSettings.xml

```diff
@@ -83,6 +83,8 @@
 			<argument index="0" name="pack" type="String">
 			</argument>
 			<description>
+			  	Loads the .pck or .zip file specified by pack in the local filesystem (res://). Returns true on success.
+			  	Note: Overrides the original file if a file shares the same path.
```

Comment:
I'm not sure I understand the note. I think you mean that the file can be reloaded over itself, usually to remove any changes made by other loaded .pck files? It might be worth trying to reword that.

Also, `specified by [code]pack[/code]`.

Reply:
If there is a .pck/.zip file loaded that contains: /script.gd and there is already a res://script.gd file, then if you would try to load res://script.gd, then it will load the one located in the .pck/.zip file

How about:
Loads the contents of the .pck or .zip file specified by [code]pack[/code] into the local filesystem (res://). Returns true on success.
Note: If a file shares the same path as a file in the local filesystem, any attempts to load that file will use the file from [code]pack[/code] instead of the original file.

Reply:
Sure, but let me offer a revised suggestion. Feel free to use whichever you feel is better, or perhaps we could go back and forth with suggestions until we find something we like:

```
Loads the contents of the .pck or .zip file specified by [code]pack[/code] into the resource filesystem (res://). Returns true on success.
Note: If a file from [code]pack[/code] shares the same path as a file already in the resource filesystem, any attempts to load that file will use the file from [code]pack[/code].
```

Reply:
I added your version, because it is better